### DISCUSSION TITLE
[Bug] Text overflows commit description in commits view

### DISCUSSION THREAD

**Discussion by SethClydesdale** [community/community#22057] [New to GitHub]
<p>Hello!</p>
<p>I wasn’t exactly sure where to report bugs for the github website, so I’ll post it here. In the commits view, if a commit description is long enough, it can overflow the container. (ex: <a href="https://github.com/SethClydesdale/genki-study-resources/commits/master" rel="nofollow noopener">expand Jul 25th’s commit</a>)</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/1fc1f485d093f1812ff9212b125e7f6843204ca0.png" data-download-href="/uploads/short-url/4wWlC46cMgrFq5wopDp08K9vK8w.png?dl=1" title="Capture" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10122549/181068731-e63c40c9-251e-4486-bdb7-9b446e115dfd.png" alt="Capture" data-base62-sha1="4wWlC46cMgrFq5wopDp08K9vK8w" width="690" height="139" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/1fc1f485d093f1812ff9212b125e7f6843204ca0_2_690x139.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/1fc1f485d093f1812ff9212b125e7f6843204ca0_2_1035x208.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/1/1fc1f485d093f1812ff9212b125e7f6843204ca0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/1fc1f485d093f1812ff9212b125e7f6843204ca0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Capture</span><span class="informations">1264×255 19.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Looking at it, it appears to be missing <code>white-space:pre-wrap;</code> which is preset on the commit description in the repo view from the class <code>.ws-pre-wrap</code>.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi <a class="mention" href="/SethClydesdale">@SethClydesdale</a>!</p>
<p>This is a good place to report these kinds of bugs, yes!</p>
<p>I believe we’ve now fixed this one - can you confirm please? If you’re still seeing it, can you give me a link?</p>

**Comment by SethClydesdale** [comment]
<p>Hi <a class="mention" href="/yamiacat">@yamiacat</a>!</p>
<p>Yes, it is indeed fixed now! If I spot anymore bugs, I’ll be sure to report them here in the future.</p>
<p>Thanks for the response and have a good weekend! :+1:</p>
